substrate/frame/multisig/src/lib.rs

```diff
@@ -108,12 +108,14 @@ where
 	depositor: AccountId,
 	/// The approvals achieved so far, including the depositor. Always sorted.
 	approvals: BoundedVec<AccountId, MaxApprovals>,
+	/// The versioned call to be executed.
+	call: VersionedCall<Call>,
```

Comment:
If this is stored it will also need a migration

Reply:
Can you explain better, please? 
I'm still new to the codebase :)

Reply:
If an item is stored and its encoding changes, then when we try to decode the value from the storage it will fails because it is a different type now.

Reply:
Hmm, I see now, can you review the pallet lottery please? @gui1117 
I still need some understanding of the migration aspect.